dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/translators/TypeTranslator.kt

```diff
@@ -119,7 +119,7 @@ internal class TypeTranslator(
     fun KaSession.buildAncestryInformationFrom(
         type: KaType
     ): AncestryNode {
-        val (interfaces, superclass) = type.getDirectSuperTypes().filterNot { it.isAny }
+        val (interfaces, superclass) = type.directSupertypes(true).filterNot { it.isAnyType }
```

Comment:
named parameter, also, AFAIU, old function used `false` here - is it expected that we change it?

Reply:
JIC it has no effect, but it might be more reliable in the case of unexpected `type`